Grids: Add AI Column API descriptions (#8121)
DevExpress/devextreme-documentation: api-reference/10 UI Components/GridBase/3 Methods/sendAIColumnRequest(columnName).md

---
---
##### shortDescription

> Sends an AI request to populate AI column values.

Reviewer: Does it simply send a request, or does it also apply the result to the column? 

```suggestion
Populates the AI column with values.
```

Author: It applies the result as well.

##### param(columnName): String
The AI column name.

---

> This method sends a request to the {WidgetName} [aiIntegration]({basewidgetpath}/Configuration/#aiIntegration) to populate AI column values.

Reviewer: ```suggestion
This method requests column values from the AI Service specified by {WidgetName} [aiIntegration]({basewidgetpath}/Configuration/#aiIntegration) settings.
```
